include/mapbox/geometry/feature.hpp

```diff
@@ -55,6 +55,10 @@ struct feature
     geometry_type geometry;
     property_map properties {};
     std::experimental::optional<identifier> id {};
+
+    // GCC 4.9 does not support C++14 aggregates with non-static data member
+    // initializers.
+    feature(geometry_type const& geometry_) : geometry(geometry_) {}
```

Comment:
This works in GCC 4.9 :+1: :
```
feature(geometry_type geometry_,
        property_map properties_ = property_map {},
        std::experimental::optional<identifier> id_ = std::experimental::optional<identifier> {})
    : geometry(std::move(geometry_)),
      properties(std::move(properties_)),
      id(std::move(id_)) {}
```

We have to be explicit when using the bracket initialization otherwise compilation fails with the following output:
> tests/test.cpp:132:42: error: converting to ‘mapbox::geometry::property_map {aka std::unordered_map<std::basic_string<char>, mapbox::geometry::value>}’ from initializer list would use explicit constructor ‘std::unordered_map<_Key, _Tp, _Hash, _Pred, _Alloc>::unordered_map(std::unordered_map<_Key, _Tp, _Hash, _Pred, _Alloc>::size_type, const hasher&, const key_equal&, const allocator_type&) [with _Key = std::basic_string<char>; _Tp = mapbox::geometry::value; _Hash = std::hash<std::basic_string<char> >; _Pred = std::equal_to<std::basic_string<char> >; _Alloc = std::allocator<std::pair<const std::basic_string<char>, mapbox::geometry::value> >; std::unordered_map<_Key, _Tp, _Hash, _Pred, _Alloc>::size_type = long unsigned int; std::unordered_map<_Key, _Tp, _Hash, _Pred, _Alloc>::hasher = std::hash<std::basic_string<char> >; std::unordered_map<_Key, _Tp, _Hash, _Pred, _Alloc>::key_equal = std::equal_to<std::basic_string<char> >; std::unordered_map<_Key, _Tp, _Hash, _Pred, _Alloc>::allocator_type = std::allocator<std::pair<const std::basic_string<char>, mapbox::geometry::value> >]’